README.md

```diff
@@ -76,6 +76,12 @@ class UsageExample
 
 ## Develop
 
+### Debug
+You can use the library with a proxy such as Fiddler to debug requests and responses as they come across the wire. Set the proxy port on the Graph object like this:
+```php
+$graph->setProxyPort("localhost:8000");
```

Comment:
Is port 8000 the standard port for Fiddler? I think it is 8888. Is there any other information here a customer may need in order to use this.

Reply:
Good question. I added some more content as well as a screenshot. Can you think of anything else to add?